File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0,a;
        for(int i = s.size()-1;i >= 0;i--){
            if(s[i] != ' ') {a = i;break;}
        }
        for(int i = a;i>=0;i--){
            if(s[i] == ' ') break;
            count++;
        }
        return count;
    }
};

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let str = s.trim();
    let i = str.length-1;
    while(i >= 0 && str[i] != ' ') {
        i--;
    }
    // console.log(str);
    return str.length-1-i;
};